app/src/main/java/com/arctouch/codechallenge/remote/api/TmdbApi.kt

```diff
@@ -0,0 +1,26 @@
+package com.arctouch.codechallenge.remote.api
+
+import com.arctouch.codechallenge.model.GenreResponse
+import com.arctouch.codechallenge.model.Movie
+import com.arctouch.codechallenge.model.UpcomingMoviesResponse
+import kotlinx.coroutines.Deferred
+import retrofit2.Response
+import retrofit2.http.GET
+import retrofit2.http.Path
+import retrofit2.http.Query
+
+interface TmdbApi {
+
+    companion object {
+        const val URL = "https://api.themoviedb.org/3/"
```

Comment:
Would you add your URL in the `BUILD.Config`? If so, why?

Reply:
Yes, my mistake
I forgot to put it in BUILD.Config
It would put to improve the architecture and facilitate the generation of version like for example (Debug and Release)